Core/Core/Dashboard/CourseCardList/Model/Entities/DashboardCard.swift

```diff
@@ -53,6 +53,17 @@ public final class DashboardCard: NSManagedObject {
         }
     }
 
+    public var isAvailableOffline: Bool {
+        guard let selections = AppEnvironment.shared.userDefaults?.offlineSyncSelections else { return true }
+        var isAvailable = false
+        selections.forEach { selection in
+            print(selection)
+            print(courseCode)
+            if selection == "courses/\(id)" { isAvailable = true }
+        }
+        return isAvailable
+    }
```

Comment:
I'm not entirely sold on this mechanism. We are hard coupling the `DashboardCard` entity with the whole `AppEnvironment` object.  Maybe we could move this logic to where we read Core Data.

Reply:
This is a temporary solution to determine the synced elements, we need a better one for sure.